_posts/2024-11-28-openloco-v24.11.md

```diff
@@ -0,0 +1,80 @@
+fixes. Please do keep identifying bugs and crashes and reporting them on Discord or on Github. The
+main reason this month has had so many fixes is due to more reports rather than last months release
+being particularly bug prone. If there is a particular bug that you know about that hasn't been 
+fixed bring up the issue on Discord there might be a reason it can't be addressed or perhaps we
+have just forgotten about it.
+
+## Add initial OpenGraphics objects (#2697)
+
+@SteveEmfore has created 7 vehicle objects for the OpenGraphics project. The OpenGraphics project
+aims to create replacements for all vanilla objects. All of the objects will have slightly
+different names and look different but the stats of the vehicles will be identical to vanilla.
+It is planned that we will incrementally improve the objects. @DazWizzle has also created some
+road vehicles for the project but they were not ready for this initial release.
+
+Next month I plan to add basic asset pack support. The plan is that when the OpenLoco asset pack
+is enabled any newly created scenarios will automatically switch objects to use the OpenGraphics
+equivalent objects. Opening a save will not cause any object switching it will only work on
+starting new scenarios. For now the default asset pack will be vanilla so there will be no
+automatic object switching.
+
+## Widget refactoring (#2674, #2675)
+
+Continuing from last month @ZehMatt has been trying to improve the GUI widgets so that they work in
+a more logical way. There is still a good bit to do on this but its slowly getting there. Once this
+work is complete we should be able to simplify how creating UI windows work making it easier to add
+new features.
+
+## Misc audio refactoring (#2699, #2700)
+
+New contributor @LeeSpork has been reworking the audio system this month. He has cleaned up and
+simplified the music playlist selection. Audio in general is one area where we have fully
+reimplemented all vanilla code so it is ripe for new features and general improvements. We look
+forward to seeing what @LeeSpork has in mind.
+
+## Finish the paint functions (#2737, #2738, #2744, #2755, #2757)
+
+@ZehMatt expressed an interest in improving how the 'paint' system worked. The one issue though
+was there was still one or two functions that hadn't been implemented preventing the improvements.
+After a quick review of what was outstanding I decided to bash out the final functions. The final
+functions were road stations, tram pylons, supports and bridges. Please be on the look out for any
+display issues with those particular areas this release.
+
+In the vanilla code the paint functions take up a huge percentage of the codebase ~20%. Its great
+milestone knowing that that is finally behind us. With this system finished expect a bunch of
+improvements in drawing over the coming months. We could add multithreading support to greatly
+improve performance, split the UI from the viewport to allow scaling the UI and viewport
```

Comment:
the comma implies there are 2 more things after it, but there is only 1 more thing so I'd just use 'and'